pkg/demoinfocs/game_events.go

```diff
@@ -256,7 +257,7 @@ func newGameEventHandler(parser *parser, ignoreBombsiteIndexNotFound bool) gameE
 		"player_given_c4":                 nil,                                   // Dunno, only present in locally recorded (POV) demos
 		"player_ping":                     nil,                                   // When a player uses the "ping system" added with the operation Broken Fang, only present in locally recorded (POV) demos
 		"player_ping_stop":                nil,                                   // When a player's ping expired, only present in locally recorded (POV) demos
-		"player_sound":                    nil,                                   // When a player makes a sound. TODO: implement player_sound
+		"player_sound":                    delayIfNoPlayers(geh.playerSound),     // When a player makes a sound
```

Comment:
I'm not sure if this is the desired behavior, but I noticed that by using `delayIfNoPlayers` we lose the very first event even though it has a player.
This made me wonder if maybe this is happening to other events as well...

Reply:
I think all the relevant events should hopefully already have `delayIfNoPlayers`